mods/voidmulti.sc2mod/base.sc2data/gamedata/actordata.xml

```diff
@@ -1483,6 +1506,7 @@
         <On Terms="AbilMorph.*.Start; MorphFrom ObserverSiegeMode; MorphTo Observer" Send="AnimBracketStop Work"/>
         <On Terms="UnitDetectedByViewer.*.On" Target="_UndetectedMuteSound" Send="SoundSetMuted"/>
         <On Terms="UnitDetectedByViewer.*.Off" Target="_UndetectedMuteSound" Send="SoundSetMuted 1"/>
+        <BarWidth value="46"/>
```

Comment:
Observer's life bar width increased from 42 to 46

Is the offset fine? I would have expected it to be changed slightly as well